### ISSUE TITLE
Header: The account menu is not announced by a screen reader, nor accessible by keyboard

### ISSUE THREAD

**Issue by jennur** [inveniosoftware/invenio-app-rdm#1281]
The items in the dropdown are not announced by the screen reader. The menu is also not fully accessible by keyboard.

<img width="197" alt="Screenshot 2022-02-17 at 16 09 20" src="https://user-images.githubusercontent.com/21052053/154510493-07d2f26b-51fc-449e-a8f1-c3d3dff882c1.png">

**Comment by lhenze** [comment]
This relates to https://github.com/inveniosoftware/invenio-app-rdm/issues/1193
